en/docs/install-and-setup/configuring-identity-server-for-ob.md

```diff
@@ -137,23 +154,23 @@ the consent page.
     cof_consent_self_link = "https://<APIM_HOST>:8243/open-banking/{version}/cbpii/"
     ```
 
-12. In the consent re-authentication step of the Accounts flow, during authorisation, the PSU is allowed to change the 
+15. In the consent re-authentication step of the Accounts flow, during authorisation, the PSU is allowed to change the 
 selected account. To enable this feature and update the account bound to the consent, set the following property to true:
 
     ``` toml
     [open_banking_uk.consent]
```

Comment:
this config is not part of berlin

Reply:
Fixed in 95d4998d1408c1aa79d8e534e6b4cabe97d99a68